npm-shrinkwrap.json

```diff
@@ -1,8 +1,17290 @@
 {
   "name": "balena-cli",
   "version": "12.40.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I believe that this change to the format and version of the shrinkwrap file is causing the build to produce the following warning and error:
```
+ node --version
v10.21.0
+ npm --version
6.14.4
+ npx check-node-version@4 --npm '^6.9.0'
npx: installed 13 in 6.391s
+ run_as npm install
+ su resin -c 'npm install'
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1,
but npm-shrinkwrap.json was generated for lockfileVersion@2. I'll try to do my best with it!
...
npm ERR! code ENOLOCAL
npm ERR! Could not install from "node_modules/etcher-sdk/unbzip2-stream@github:balena-io-modules/unbzip2-stream#942fc218013c14adab01cf693b0500cf6ac83193" as it does not contain a package.json file.
```
I was able to reproduce the build warning and error on my laptop with:
```
$ rm -rf node_modules
$ npm install
$ npm run package
```
It happens with this PR's branch, but does not happen with the master branch. I gather that lockfileVersion 2 is produced by npm version 7: https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json

It is described as backwards compatible but perhaps not sufficiently so. :-) The CLI only supports Node v10 and v12, and I suggest using the versions of npm that "ship with" those versions of Node.js -- currently npm v6.14.11:
```
$ nvm use 10
Now using node v10.23.3 (npm v6.14.11)

$ nvm use 12
Now using node v12.20.2 (npm v6.14.11)
```
After switching to npm v6.14.11, the shrinkwrap file may be reverted and fixed with:
```
$ git checkout master -- npm-shrinkwrap.json
$ rm -rf node_modules
$ npm install
```